SpreadsheetStreams/Code/Excel/ExcelSpreadsheetWriter.cs

```diff
@@ -14,15 +14,15 @@ public class ExcelSpreadsheetWriter : SpreadsheetWriter
     {
         #region Constructors
 
-        public ExcelSpreadsheetWriter(Stream outputStream, CompressionLevel compressionLevel = CompressionLevel.Fastest)
+        public ExcelSpreadsheetWriter(Stream outputStream, CompressionLevel compressionLevel = CompressionLevel.Fastest, bool leaveOpenAfterFinish = false)
```

Comment:
Naming of the new parameter is inconsistent. Please keep it `leaveOpen`. That's how it's called in ZipArchive and it's very clear :-)

Reply:
ok, no problem. I just thought that user of this library does not need to know how `ZipArchive` works (or even that zip is used at all )